charts/kafkametriccollectoragent/templates/configmap.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "kafkametriccollectoragent.fullname" . }}-config-cm
+  namespace: {{ .Release.Namespace }}
+data:
+  application.yml: |-
```

Comment:
Can we just define application.yml and use it from values.yml?
We can reference the whole configuration block.